File: programowanie i podstawy algorytmiki/V zajecia/cpp/pos.cpp
#include <bits/stdc++.h>
using namespace std;

int suma_liczb(const vector<int>& liczby) {
  int suma = 0;
  for (int x : liczby)
    suma += x;
  return suma;
}

vector<int> wektor_cyfr(long long x) {
  vector<int> cyfry;
  for (int i = 0; i < 19; i++) {
    cyfry.push_back(x % 10);
    x /= 10;
  }
  reverse(cyfry.begin(), cyfry.end());
  return cyfry;
}

vector<int> ustaw(const vector<int>& cyfry_a, const vector<int>& cyfry_b) {
  vector<int> cyfry_c = cyfry_b;
  bool dziewiatki = false;
  for (int i = 0; i < 19; i++) {
    if (dziewiatki)
      cyfry_c[i] = 9;
    else if (cyfry_a[i] != cyfry_b[i]) {
      cyfry_c[i] = cyfry_b[i] - 1;
      dziewiatki = true;
    }
  }
  return cyfry_c;
}

int main() {
  ios_base::sync_with_stdio(false);
  cin.tie(0);

  long long a, b;
  cin >> a >> b;
  vector<int> cyfry_a = cyfry_liczby(a), cyfry_b = cyfry_liczby(b);
  vector<int> cyfry_c = popraw(cyfry_a, cyfry_b);
  int wynik1 = suma_liczb(cyfry_b);
  int wynik2 = suma_liczb(cyfry_c);
  cout << max(wynik1, wynik2) << "\n";

  return 0;
}

File: programowanie i podstawy algorytmiki/V zajecia/python/pos.py
# def suma_cyfr(x):
#     if x == 0:
#         return 0
#     return (x % 10) + suma_cyfr(x // 10)
#
#
# a, b = map(int, input().split())
# # lista = []
# # for x in range(a, b+1):
# #     list.append(suma_cyfr(x))
# # print(max(list))
# print(max([suma_cyfr(x) for x in range(a, b+1)]))


# def suma_cyfr(x):
#     if x == 0:
#         return 0
#     return (x % 10) + suma_cyfr(x // 10)
#
# def cyfra_znaczaca(x):
#     if x >= 10:
#         return cyfra_znaczaca(x // 10)
#     return x
#
# def liczba_cyfr(x):
#     if x == 0:
#         return 0
#     return 1 + liczba_cyfr(x // 10)
#
#
# a, b = map(int, input().split())
# wynik1 = suma_cyfr(b)
# wynik2 = (cyfra_znaczaca(b) - 1) + 9 * (liczba_cyfr(b) - 1)
# print(max(wynik1, wynik2))


def lista_cyfr(x):
    cyfry = []
    for _ in range(19):
        cyfry.append(x % 10)
        x //= 10
    return cyfry[::-1]


def ustaw(cyfry_a, cyfry_b):
    cyfry_c = list(cyfry_b)
    dziewiatki = False
    for i in range(19):
        if dziewiatki:
            cyfry_c[i] = 9
        elif cyfry_a[i] != cyfry_b[i]:
            cyfry_c[i] = cyfry_b[i] - 1
            dziewiatki = True
    return cyfry_c


a, b = map(int, input().split())
cyfry_a, cyfry_b = lista_cyfr(a), lista_cyfr(b)
cyfry_c = ustaw(cyfry_a, cyfry_b)
wynik1 = sum(cyfry_b)
wynik2 = sum(cyfry_c)
print(max(wynik1, wynik2))